setup.cfg

```diff
@@ -11,7 +11,6 @@ ignore =
     Makefile.configure.Darwin
     zc.recipe.egg_/*
     zc.recipe.egg_
-    tox.ini
     .github/*
     .github
```

Comment:
With check-manifest >= 0.42 you may have to change `.github/*` to `.github/**`, to ignore files in subdirectories recursively.  (The two-* version also works with check-manifest < 0.42.)